ext/websocket/01_websocket.js

```diff
@@ -440,6 +446,7 @@
             const event = new MessageEvent("message", {
               data,
               origin: this[_url],
+              [_skipInternalInit]: true,
```

Comment:
This key symbol is observable and leakable via `Proxy`. I still don't think this is a good idea. This should be a factory method.